index.html

```diff
@@ -279,7 +284,46 @@ <h3 class="down under">Frameworks<span class="greycol"> > </span></h3>
           <h3 class="down under">Skills<span class="greycol"> > </span></h3>
         </div>
       </section>
+      <section id="contact-me">
+        <div class="contact-wrap">
+          <h3>Contact Me</h3>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <form action="https://formspree.io/f/myyawdlo" method="POST">
+            <input
+              class="input"
+              type="text"
+              name="name"
+              placeholder="Enter Name"
+              required
+              maxlength="30"
+            />
+            <input
+              class="input"
+              type="email"
+              name="email"
+              placeholder="Enter Email"
+              required
+            />
+            <textarea
+              class="input"
+              name="message"
+              id=""
+              cols="30"
+              rows="10"
+              placeholder="Enter message here"
+              maxlength="500"
+              required
+            ></textarea>
+            <button type="submit" class="see-project">Get In Touch</button>
+          </form>
+          <hr />
+        </div>
+      </section>
```

Comment:
- [OPTIONAL] Kindly remove the white space at the bottom of the form on your page. Nothing like that in the Figma Design.

- [OPTIONAL] I noticed some missing design from your background image used as oppose to how there are on the Figma design. I suggest you check the image you used for the background image if it is the correct one. Kindly look at the red circle I made on your design and the Figma design to get what I am talking about. 

### Your Design
![your-form-section](https://user-images.githubusercontent.com/26597432/220442176-fb158f28-595d-4ef8-8c72-7d232cf563a3.png)

### Figma Design 
![figma-form-section](https://user-images.githubusercontent.com/26597432/220442225-3b8e8c60-4514-4307-be69-41e80fda75af.png)